src/transformers/models/detr/image_processing_detr_fast.py

```diff
@@ -336,29 +319,28 @@ class DetrImageProcessorFast(BaseImageProcessorFast):
             The size `{"height": int, "width" int}` to pad the images to. Must be larger than any image size
             provided for preprocessing. If `pad_size` is not provided, images will be padded to the largest
             height and width in the batch.
-    """
-
+    """,
+)
+class DetrImageProcessorFast(BaseImageProcessorFast):
+    resample = PILImageResampling.BILINEAR
+    image_mean = IMAGENET_DEFAULT_MEAN
+    image_std = IMAGENET_DEFAULT_STD
+    format = AnnotationFormat.COCO_DETECTION
+    do_resize = True
+    do_rescale = True
+    do_normalize = True
+    do_pad = True
```

Comment:
@yonigozlan Hi Yoni, what is the rule for define default parameter in here. For example I'm confused that why does `recalse_factore = 1/255` is not written here while it has defined int both `BaseImageProcessorFast`